### PR TITLE
Prepare release

### PR DESCRIPTION
**Pull request by Yajo**

### CODE REVIEW

**Comment by carlosecv** [comment]
O my God!!! I hope can follow this rhythm.
 Excelent Work!

File: copier.yml [hunk 1]
```diff
@@ -213,16 +234,6 @@ domains_prod:
...
-    # Main domains, with TLS and cert resolved with "letsencrypt" resolver
-    - hosts: [www.main.com, shop.main.com]
-    # Alternate domains that redirect to main ones
-    - {hosts: [main.com, old.main.com], redirect_to: www.main.com}
-    - {hosts: [www.shop.main.com, old.shop.main.com], redirect_to: shop.main.com}
-    # VPN-only domain, with self-signed certificate
-    - {hosts: [vpn.main.com], cert_resolver: true}
-
     ⚠ Remember that you have to write one-line YAML, at least until Copier
     supports multiline input
```
Comment (joao-p-marques) [inline copier.yml:238]: Wouldn't this comment become deprecated if we are using `multiline`?

Comment (Yajo) [reply copier.yml:238]: Indeed.

Comment (Yajo) [reply copier.yml:238]: Oh sorry, copier supports multiline only since the latest alpha, so this is still useful until a v6 is released.

**Review by joao-p-marques — APPROVED** [review]

**Pull request merged**